quickstarts/about-quickstarts.md

```diff
@@ -1,37 +1,107 @@
-# The quickstart tutorial template
 
-The quickstart or `Hello World!` tutorial is for introducing your users to your product or framework for the first time. Users can typically complete these tutorials in less than 15 minutes. It's task-based and installation-focused.
+# Quickstart Guide
 
-## About the overview section
+{Read this document before you start working on the [quickstart template](template-quickstarts.md)}
 
-Use this section to provide a short description of your product and its purpose, and to describe what the user will accomplish in this tutorial.
+## Introduction
 
-You can also state the intended audience for your tutorial and mention any concepts users should be familiar with before starting the tutorial. 
+The quickstart is intended for introducing your users to your product or framework for the first time. 
+It focuses on the **hero feature** of the product and helps your users to start using the product as quickly as possible.
 
-## About the installation section
+A good quickstart answers the following questions:
 
-<!--link out to installation guide unless it's mostly links-->
+* What can this software/application do? Purpose of this quickstart
+* Where can I find the product. It can point to a web address or a download link.
+* How can I get any required access keys or authentication credentials?
+* How can I install/setup this software?
+* How can I configure the product? For a developer-focused product, this usually involves creating a hello world application. For an end-user product, this may mean setting up a corporate or user profile and choosing other preferences.
+* How can I get started with the product? This is often a walkthrough of a simple but common use case, that helps the user learn the patterns of the product
+* What other options (advanced features) are available to explore more in the product?
+* What should I do if something does not work? (Link to troubleshooting guide or any support email ID)
 
```

Comment:
I'd suggest adding something like:

A quickstart is often the first opportunity for your user to build an opinion about the technical quality of your project. Hence, the quickstart should:
* Guarantee success for anyone who follows the quickstart steps.
* Work with any application version the user might use.
* Help the user get excited about the application.

Reply:
We have added the following considering your suggestion:

A quickstart is often the first opportunity for your user to build an opinion about the technical quality of your project. Hence, the quickstart should:
- Make users comfortable using the application.
- Work with any application version the user might use.
- Lead to the willingness to handle complex workflows.
- Help the users in exploring additional features.
- Improve customer experience and help to reduce costs by lowering the number of support requests.